README.md

```diff
@@ -76,6 +76,9 @@ Otherwise, installation of `modin[mpi]` may fail. Refer to
 [Installing with pip](https://unidist.readthedocs.io/en/latest/installation.html#installing-with-pip)
 section of the unidist documentation for more details about installation.
 
+**Note:** Since Modin 0.30.0 we use a reduced set of Ray dependencies: ray-core instead of ray-default.
+This means that the dashboard and cluster launcher are no longer installed by default.
```

Comment:
```suggestion
**Note:** Since Modin 0.30.0 we use a reduced set of Ray dependencies: `ray[core]` instead of `ray[default]`.
This means that the dashboard and cluster launcher are no longer installed by default.
If you need those, consider installing `ray[default]` along with `modin[ray]`.
```
Should we also add this note for intallation with conda?